### PR TITLE
helix: fix libc++_shared not found

### PR DESCRIPTION
**Pull request by MrAdityaAlok**
- fix syntax highlighting not working due to `libc++_shared` being linked but not found at runtime.

@xeffyr  is there another way to fix this? I do not know why `libc++_shared` is not found when built on `ci` but works well when built on termux.

### CODE REVIEW

**Comment by xeffyr** [comment]
The clang package in Termux is patched to enforce setting runpath without passing any flags manually. On CI we use official NDK toolchain and to set runpath passing some flags is necessary. If your package ignores LDFLAGS variable, there will be issues.

File: packages/helix/build.sh [hunk 1]
```diff
@@ -1,23 +1,43 @@
...
-	mkdir -p ${TERMUX_PREFIX}/lib/helix
-	cp -r runtime ${TERMUX_PREFIX}/lib/helix
-	install -Dm755 -t ${TERMUX_PREFIX}/lib/helix target/${CARGO_TARGET_NAME}/release/hx
+	install -Dm755 -t "${TERMUX_PREFIX}/lib/helix" "target/${CARGO_TARGET_NAME}/release/hx"
+	install -Dm666 -t "${TERMUX_PREFIX}/lib/helix" languages.toml
+}
+
+termux_step_create_debscripts() {
+	cat > postinst <<- EOF
+		#!${TERMUX_PREFIX}/bin/bash
```
Comment (xeffyr) [inline packages/helix/build.sh:35]: Remove redundant indentation from heredoc.

Should be like there:
```
cat > postinst <<- EOF
#!${TERMUX_PREFIX}/bin/bash
...
```

Same issue with heredoc in make_install step.

File: packages/helix/build.sh [hunk 2]
```diff
@@ -1,23 +1,43 @@
...
+
+		if ! test -f "${TERMUX_ANDROID_HOME}/.config/helix/languages.toml"; then
+			mkdir -p "${TERMUX_ANDROID_HOME}/.config/helix"
+			mv "${TERMUX_PREFIX}/lib/helix/languages.toml" "${TERMUX_ANDROID_HOME}/.config/helix"
```
Comment (xeffyr) [inline packages/helix/build.sh:39]: Use `cp`.

**Review by xeffyr — CHANGES_REQUESTED** [review]

File: packages/helix/build.sh (hunk 1 above)
Comment (MrAdityaAlok) [reply packages/helix/build.sh]: Removed

File: packages/helix/build.sh (hunk 2 above)
Comment (MrAdityaAlok) [reply packages/helix/build.sh]: Done

**Comment by MrAdityaAlok** [comment]
@xeffyr Thank you for your help and now it is ready to be merged.

**Review by xeffyr — APPROVED** [review]

**Pull request merged**